proposals/ecosystem/ecosystem.shex

```diff
@@ -87,3 +87,44 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 <#ApplicationShape> {
   a [ eco:Application ] ;
 }
+
+################################################################
+# Access Need Groups
+################################################################
+
+# Validate an instance of the eco:AccessNeedGroup class
+<#AccessNeedGroupShape> {
+  a [ eco:AccessNeedGroup ] ;
+  eco:accessNecessity [ eco:AccessRequired eco:AccessOptional ] ;
+  eco:authenticatesAs IRI ;
+  eco:requestsAccess IRI+ ;
+  eco:hasAccessDecoratorIndex IRI
+}
+
+# Validate an instance of a prepared eco:AccessNeedGroup class
+<#PreparedAccessNeedGroupShape> {
+  a [ eco:AccessNeedGroup ] ;
+  eco:accessNecessity [ eco:AccessRequired eco:AccessOptional ] ;
+  eco:authenticatesAs IRI ;
+  eco:requestsAccess IRI+ ;
+  eco:hasAccessDecoratorIndex IRI ;
+  eco:needGroupLabel xsd:string ;
+  eco:needGroupExplanation xsd:string
+}
+
+
+# Validate an instance of the eco:AccessNeed class
+<#PreparedAccessNeedShape> {
+  a [ eco:AccessNeed ] ;
+  eco:inAccessNeedGroup IRI+ ;
```

Comment:
this very likely could use `^eco:requestsAccess IRI ;` and not rely on defining inverse property
https://shex.io/shex-primer/#inverse-properties

ping @ericprud

Reply:
I'm certainly in favor of having using only one direction or the other. WRT which, I guess I like the arcs-in (`<<group>> eco:includesNeed <<need>>`) better 'cause we don't have to invent URLs for all of the needs.

I used `includesNeed` rather than `requestsAccess` 'cause we currently have a distinction between root rules and overrides. I want to play with a few use cases to figure out whether they can be the same thing so in the mean time, `includesNeed` preserves the current expressivity.